ls1intum/Artemis
# [Chore] Reformat to Prettier 2.3.0

### Motivation and Context
Currently the client style check is failing due to the upgrade to prettier 2.3.0 which features several styling improvements. Most notably these two changes affect the repository:
https://prettier.io/blog/2021/05/09/2.3.0.html#dont-print-parens-around-nested-type-assertions-1[…]erprettierpull10702-by-thorn0httpsgithubcomthorn0
https://prettier.io/blog/2021/05/09/2.3.0.html#prevent-wrapping-object-properties-with-short-key[…]erprettierpull10335-by-thorn0httpsgithubcomthorn0

### Description
I ran `yarn lint:fix` and `yarn prettier:format`

### Steps for Testing
1. Run `yarn lint:fix` and `yarn prettier:format`
2. Verify the result is the same

### Test Coverage
Unchanged
